static/src/javascripts/bootstraps/media.js

```diff
@@ -214,8 +214,13 @@ define([
                 // preroll for videos only
                 if (mediaType === 'video') {
 
+                    if (showEndSlate && detect.isBreakpoint({ min: 'desktop' })) {
+                        initEndSlate(player, endSlateUri);
+                    }
+
                     player.fullscreener();
-                    // Init plugins
+
+                    // do ads last so if the ad blocker throws an exception it doesn't stop anything else
```

Comment:
try/catch performance these days is a misnomer. We use it all over the shop now, including the invocation of this module (via wrapping in raven.js) so don't worry.